test.js

```diff
@@ -164,4 +174,50 @@ describe('remark-lint-no-dead-urls', () => {
       expect(vFile.messages.length).toBe(0);
     });
   });
+
+  describe('skipLocalhost: true', () => {
+    test('localhost', () => {
+      const lint = processMarkdown(
+        dedent`
+							- [http://localhost](http://localhost)
```

Comment:
This indentation looks pretty extreme 😬

Reply:
Ooops, that's prettier config with tabs))